submarine-security/spark-security/pom.xml

```diff
@@ -572,6 +572,20 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>spark-3.2</id>
+      <properties>
+        <spark.version>3.2.0</spark.version>
+        <scala.version>2.12.14</scala.version>
```

Comment:
why scala.version here is 2.12.14 instead of 2.12.15

Reply:
I agree with @akizminet. This should be 2.12.15 to match line #51.

Reply:
Done